### PR TITLE
Update Admin Notify

### PR DESCRIPTION
**Pull request by yesszus**
Add a new and missing lang codes.

### CODE REVIEW

File: e107_languages/English/admin/lan_notify.php [hunk 1]
```diff
@@ -18,15 +22,22 @@
...
 define("NS_LAN_2", "IP banned for flooding site");
+define("NS_LAN_3", "IP banned for multiple failed login attempts");
 
 define("NN_LAN_1", "News Events");
 define("NN_LAN_2", "News item submitted by user");
 define("NN_LAN_3", "News item posted by admin");
 define("NN_LAN_4", "News item edited by admin");
 define("NN_LAN_5", "News item deleted by admin");
+define("NN_LAN_6", "News notification triggered");
+define("NN_LAN_7", "News");
```
Comment (CaMer0n) [inline e107_languages/English/admin/lan_notify.php]: I'm pretty sure this is a duplicate. ?

Comment (yesszus) [inline e107_languages/English/admin/lan_notify.php]: I'm not find this duplicates.

**Comment by yesszus** [comment]
@CaMer0n u find duplicates?

File: e107_languages/English/admin/lan_notify.php (hunk 1 above)
Comment (CaMer0n) [inline e107_languages/English/admin/lan_notify.php]: news_lan

Comment (yesszus) [inline e107_languages/English/admin/lan_notify.php]: Ok, changed!
Removed define("NN_LAN_7", "News");

**Comment by tgtje** [comment]
Just a question/remark : i was under the impression (mind.. i am not a coder) that a key (for an array) may not be a LAN ?

**Comment by Moc** [comment]
Correct, this needs to be corrected before the changed can be merged in. 

For example 'news' and 'administrators' should not be translated.

**Comment by yesszus** [comment]
@CaMer0n pls review this.

**Pull request closed without merging**